<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>firstWidget</class>
 <widget class="QWidget" name="firstWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
*{
color: #78799c;
border: none;
}

#firstWidget, #frame_2, #frame_5{
	background-color: #2a2c49;
}

#mainBody,#Form{
	background-color: #1a1f39;
}

QLineEdit{
	background-color:transparent;
}

#searchFrame, QLineEdit{
	border-radius: 10px;
	border: 2px solid #78799c;
}

#appHeader{
	color:#78799c;
}

#prodottiHeader{
	color:  #a06ce3;
}

#avantiarw{
	border: 1px solid #a06ce3;
	background-color: #a06ce3 ;
	border-radius: 5px;
	
}

#finestreSecondarie{
	background-color: #2a2c49;
	border: solid;
	border-radius: 20px;
}

QLabel{
	color:#78799c;
}

#mainFrame{
	background-color: #2a2c49;
}

#homeBtn{
	color:#78799c;
	background-color:#1a1f39;
	padding : 10px 5px;
	text-align:left;
	border-top-left-radius:25px;
}

QPushButton{
	padding : 10px 5px;
	text-align:left;
	color:#78799c;
	font: 10pt &quot;Leelawadee UI&quot;;
}


#frame_3{
	background-color:#2a2c49;
	border: solid;
	border-radius: 20px;
}
QTableWidget{
	background-color:#2a2c49;
	border: solid;
}
#leftMenu,#frame{
background-color: #2a2c49;
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="leftMenu" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>0</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_5">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>54</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>68</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>15</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="homeBtn">
              <property name="font">
               <font>
                <family>Leelawadee UI</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Home</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/purpleIcons/assets/#78799c(purple) icons/home.svg</normaloff>:/purpleIcons/assets/#78799c(purple) icons/home.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="iMieiProdottiBtn">
              <property name="font">
               <font>
                <family>Leelawadee UI</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>I miei Prodotti</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/purpleIcons/assets/#78799c(purple) icons/trending-up.svg</normaloff>:/purpleIcons/assets/#78799c(purple) icons/trending-up.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="iMieiDatiBtn">
              <property name="font">
               <font>
                <family>Leelawadee UI</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>I miei dati</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/purpleIcons/assets/#78799c(purple) icons/shopping-cart.svg</normaloff>:/purpleIcons/assets/#78799c(purple) icons/shopping-cart.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignBottom">
             <widget class="QPushButton" name="infoBtn">
              <property name="font">
               <font>
                <family>Leelawadee UI</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Info</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/purpleIcons/assets/#78799c(purple) icons/info.svg</normaloff>:/purpleIcons/assets/#78799c(purple) icons/info.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mainBody" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignTop">
       <widget class="QWidget" name="mainFrame" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignLeft|Qt::AlignTop">
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="openLeftMenu">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>icons/menu.svg</normaloff>icons/menu.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="applicationName">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt; font-weight:600;&quot;&gt;Usato&lt;/span&gt;&lt;span style=&quot; font-size:28pt; font-weight:600; color:#aaaaff;&quot;&gt;Beato&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
           <zorder>applicationName</zorder>
           <zorder>openLeftMenu</zorder>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter|Qt::AlignTop">
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_4"/>
          </widget>
         </item>
         <item alignment="Qt::AlignRight|Qt::AlignTop">
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="openRightMenu">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/purpleIcons/assets/#78799c(purple) icons/user.svg</normaloff>:/purpleIcons/assets/#78799c(purple) icons/user.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_toPaste">
         <property name="leftMargin">
          <number>40</number>
         </property>
         <property name="topMargin">
          <number>40</number>
         </property>
         <property name="rightMargin">
          <number>40</number>
         </property>
         <property name="bottomMargin">
          <number>40</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="rightMenu" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>0</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignTop">
       <widget class="QFrame" name="frame_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>11</number>
         </property>
         <item>
          <widget class="QPushButton" name="quitBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Leelawadee UI</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Quit</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/purpleIcons/assets/#78799c(purple) icons/x-octagon.svg</normaloff>:/purpleIcons/assets/#78799c(purple) icons/x-octagon.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
